src/pl/senla/hotel/configuration/AppConfiguration.java

```diff
@@ -0,0 +1,70 @@
+package pl.senla.hotel.configuration;
+
+import java.util.Properties;
+
+import static pl.senla.hotel.constant.PropertiesConstant.*;
+
+public class AppConfiguration implements Configuration{
+
+    private static AppConfiguration appConfiguration;
+    private final Properties properties;
+
+    private AppConfiguration() {
+        PropertiesLoader propertiesLoader = new AppPropertiesLoader();
+        properties = propertiesLoader.loadConfiguration();
+    }
+
+    public static AppConfiguration getAppConfiguration() {
+        if (appConfiguration == null) {
+            appConfiguration = new AppConfiguration();
+        }
+        return appConfiguration;
+    }
+
+    @Override
+    public Boolean getValueIsAbleToChangeRoomStatus() {
```

Comment:
> You did not need separate method to each property. In long-term development, the number of properties can exceed hundreds. In this case, your class will be huge. You just need to write general methods for each type passing the name of the property, as I already suggested in some comment

Aleh, should I use this in appropriate places in program?
    pl.senla.hotel.service.ServiceRoomImpl (for example here)
    Boolean.valueOf(configuration.getProperties().getProperty(KEY_ABLE_TO_CHANGE_ROOM_STATUS))

instead of reading of each property by it's key in the same places using get-methods for this property from properties-file which I got in Main: 
    configuration.getValueIsAbleToChangeRoomStatus()
 
Did I understand correctly?

Reply:
![image](https://github.com/dudarenko-dmitry/senla_training/assets/102149674/97b8efd5-98ff-4602-ad8e-7566311644a2)

Reply:
Not certainly in that way. Your property class should not only store your property files, but also give them in a convenient form so that you don’t have to get them through the internal fields of the object. Ideally the usage should look something like this:

```
if(configuration.getBooleanProperty(KEY_ABLE_TO_CHANGE_ROOM_STATUS)) {
   ...
}
```

Reply:
And property class in that case only need few methods

![изображение](https://github.com/dudarenko-dmitry/senla_training/assets/31473268/e674cb5f-21d9-4f57-a119-fee403567bb2)

Reply:
... and I should implement such changes:
![image](https://github.com/dudarenko-dmitry/senla_training/assets/102149674/6c77af2f-3931-42b2-b14c-fee76e075bfc)

Reply:
I got it.
Everithing will be fixed in 10 min